src/workloads/sharding/MultiShardTransactionsWithManyNamespaces.yml

```diff
@@ -0,0 +1,237 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+
+GlobalDefaults:
+  Database0: &Database0 databaseForMultiShardTestingIndex0
+  Database1: &Database1 databaseForMultiShardTestingIndex1
+
+  ApproxDocumentSize: &ApproxDocumentSize 2000  # = 2kB
+  ApproxDocumentSize50Pct: &ApproxDocumentSize50Pct 1000  # = 1kB
+  DocumentCount: &DocumentCount 100_000
+
+  MaxPhase: &MaxPhase 4
+
+  RandomIdFilter: &RandomIdFilter {_id: {^RandomInt: {min: 1, max: *DocumentCount}}}
+
+  UpdateOperation: &UpdateOperation
+    OperationName: updateOne
+    OperationCommand:
+      Filter: *RandomIdFilter
+      Update: {$inc: {counter: 1}}
+
+  UpdateOperationDatabase0: &UpdateOperationDatabase0
+    <<: *UpdateOperation
+    Database: *Database0
+
+  UpdateOperationDatabase1: &UpdateOperationDatabase1
+    <<: *UpdateOperation
+    Database: *Database1
+
+Actors:
+- Name: CreateShardedCollection
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            enableSharding: *Database0
+        - OperationName: AdminCommand
+          OperationCommand:
+            enableSharding: *Database1
+        - OperationName: AdminCommand
+          OperationCommand:
+            movePrimary: *Database0
+            to: rs0
+        - OperationName: AdminCommand
+          OperationCommand:
+            movePrimary: *Database1
+            to: rs1
+
+- Name: LoadInitialData
+  Type: MonotonicLoader
+  Threads: 4
+  Phases:
+  - &Nop {Nop: true}
+  - Repeat: 1
+    Threads: 4
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    CollectionCount: 25
+    Database: *Database0
+    Document: &Document
+      counter: 0
+      padding: {^FastRandomString: {length: {^RandomInt: {min: *ApproxDocumentSize50Pct, max: *ApproxDocumentSize}}}}
+  - Repeat: 1
+    Threads: 4
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    CollectionCount: 25
+    Database: *Database1
+    Document: *Document
+  - *Nop
+  - *Nop
+
+- Name: QuiesceDatabase0
```

Comment:
[question] Could you explain the reason for doing this? I saw the header comment [here](https://github.com/mongodb/genny/blob/54e18a79d2836675e1a9f967c4cea2ca15f46991/src/cast_core/include/cast_core/actors/QuiesceActor.hpp#L31) but I'm still unclear on that.

Reply:
Basically it does fsync for all cluster after we are finished writing data and before we start querying it to make sure the state of the cluster is more consistent between runs by the time we start executing queries.

Reply:
The fsync command forces the mongod process to flush all pending writes to the storage layer